From 2c7ec18f73a1d3193e6ba4ecc57b27540350dbc6 Mon Sep 17 00:00:00 2001
From: Gabriele M <moto.falcon.git@gmail.com>
Date: Tue, 17 Jan 2017 13:51:29 +0100
Subject: [PATCH 4/7] Allow to exclude imgs from target-files zip

imgs can be generated from the files that are included in the zip, so
allow to exclude them to reduce the target-files zip size. This could
also speed up the signing process when sign_target_files_apks is used.
This script always drops the images included in the target-files zip,
so excluding them would make absolutely no difference and we would
not waste time creating and zipping useless files.

Change-Id: Ic40cfda3975f28e98a2dfa6d61fd8ed2b9f4b20d
---
 core/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/Makefile b/core/Makefile
index 2b8a65c24..1555aca46 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -2719,8 +2719,10 @@ ifdef BUILT_VENDOR_MATRIX
 endif
 endif
 
+ifneq ($(TARGET_FILES_NO_IMG),true)
 	$(hide) PATH=$(foreach p,$(INTERNAL_USERIMAGES_BINARY_PATHS),$(p):)$$PATH MKBOOTIMG=$(MKBOOTIMG) \
 	    ./build/tools/releasetools/add_img_to_target_files -a -v -p $(HOST_OUT) $(zip_root)
+endif
 	@# Zip everything up, preserving symlinks and placing META/ files first to
 	@# help early validation of the .zip file while uploading it.
 	$(hide) find $(zip_root)/META | sort >$@.list
-- 
2.17.1

